rest-server/admin/src/main/scala/com/actionml/admin/MongoAdministrator.scala

```diff
@@ -33,11 +33,17 @@ class MongoAdministrator extends Administrator with JsonParser {
   private val storage = MongoStorage.getStorage("harness_meta_store", codecs = List.empty)
 
   private lazy val enginesCollection = storage.createDao[Document]("engines")
-  @volatile private var engines = Map.empty[EngineId, Engine]
+  @volatile private var engines = Map.empty[String, Engine]
 
   drawActionML
   private def newEngineInstance(engineFactory: String): Engine = {
     Class.forName(engineFactory).newInstance().asInstanceOf[Engine]
```

Comment:
@andreyshikov We need to replace this with the companion object factory/builder invocation.